pdc_client/__init__.py

```diff
@@ -162,11 +162,10 @@ def decode(req):
         content_type = "application/json"
         encode = json.dumps
         self.client = beanbag.BeanBag(url, session=self.session, fmt=(content_type, encode, decode))
-        if not develop:
-            # For develop environment, we don't need to require a token
-            if not token:
-                token = self.obtain_token()
-            self.session.headers["Authorization"] = "Token %s" % token
+        # NOTE: CSRF token can be required even in development environment.
```

Comment:
Actually I am not quite understand why we want to use token in devlop.
And I test it in my local env, it will fail with 401.
I am wondering whether you are configuring some maybe wsgi module in your apache's config, or sth else.
But I guess that should be inconvenient for the developer.